src/index.scss

```diff
@@ -1,5 +1,22 @@
 // Import Normalize Stylesheet
-@import 'styles/normalize.scss';
+@import 'styles/normalize';
 
-// Import Typography Stylesheet
-@import 'styles/utilities/typography.scss';
+// Import Utilities
+@import
+  'styles/utilities/alignment',
+  'styles/utilities/breakpoints',
+  'styles/utilities/colors',
+  'styles/utilities/fonts',
+  'styles/utilities/typography';
+
+// Import pages
+// TODO: need to add pages import
+
+// Import Sections
+// TODO: refactor this though
+@import
+  'sections/DonateSectionOne/DonateSectionOne',
+  'sections/DonateSectionTwo/DonateSectionTwo';
+
+// Import components
+@import 'components/components';
```

Comment:
Do we want to import at this level? For pages, sections and components it might make more sense to have the import at component level.

Reply:
Open to opinions though, either works for me

Reply:
Well sections may get x'd cause I was able to get a working way of doing a page builder. If you login to wp you'll see ACF PRO - I was able to get a "free" one